src/components/delete/mod.rs

```diff
@@ -0,0 +1,49 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod component;
+mod sign_as;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = crate::GlobalContext)]
+#[interactive_clap(output_context = DeleleteComponentsFromAccountContext)]
+pub struct DeleleteComponentsFromAccount {
```

Comment:
Fix the typo in `DeleleteComponentsFromAccount` -> `DeleteComponentsFromAccount` (and in all the other places below, including context name)

Reply:
Done